### PR TITLE
Cifar10 - fp16 + convergence tests

### PR DESCRIPTION
**Pull request by bearpelican**
Attempt to speed up cifar10 training: cifar10/cifar10-convergence-playground.ipynb
* Basically try to use high learning rates + CLR and beat our original cifar10 benchmark
* The paper (listed in the notebook) has some great resources on hyperparameter tuning for cifar10
* One improvement is to use cyclical momentum - inverse proportional to learning rate increase

Time profiling fp16 model conversion: fp16/cifar10-fp16-speed-test.ipynb
* Doesn't seem to speed up anything so far. Added comments at the bottom of the notebook
* It's also weird that training on 32x32 size images isn't any slower than 8x8. Will need to profile the dataloader

### CODE REVIEW

**Comment by bearpelican** [comment]
@brettkoonce
